library/std/src/sys/windows/process.rs

```diff
@@ -621,6 +623,10 @@ impl Process {
         unsafe { c::GetProcessId(self.handle.as_raw_handle()) as u32 }
     }
 
+    pub fn main_thread_handle(&self) -> BorrowedHandle<'_> {
+        unsafe { BorrowedHandle::borrow_raw(self.main_thread_handle.as_raw_handle()) }
```

Comment:
```suggestion
        self.main_thread_handle.as_handle()
```

The [`AsHandle`](https://doc.rust-lang.org/std/os/windows/io/trait.AsHandle.html) trait does this for us.